packages/front-end/components/FactTables/ColumnModal.tsx

```diff
@@ -56,7 +59,19 @@ export default function ColumnModal({ existing, factTable, close }: Props) {
             name: value.name,
             numberFormat: value.numberFormat,
             datatype: value.datatype,
+            alwaysInlineFilter: value.alwaysInlineFilter,
           };
+
+          if (
+            data.alwaysInlineFilter &&
```

Comment:
Should we warn if existing.alwaysInlineFilter and you change the datatype from string AND it's used on any metrics? I know it keeps working, but that is kind of good luck. If you actually change the column data type you probably don't *want* it to keep working as an inline filter!

Reply:
It's such an edge case, I wouldn't worry about it.